src/cms/templates/users/region/list_row.html

```diff
@@ -26,11 +28,13 @@
 			{% endif %}
         </a>
 	</td>
-	<td class="pl-2">
+	<td class="px-2">
 		{% if user.is_active %}
 		<i data-feather="check"></i>
 		{% else %}
-		<i data-feather="x"></i>
+		<a href="{% url 'resend_activation_link_region' region_slug=region.slug user_id=user.pk %}" class="hover:underline text-blue-500 hover:text-blue-600 whitespace-nowrap">
+			{% trans 'Resend activation link' %}
+		</a>
```

Comment:
```suggestion
			<i data-feather="x"></i>
			<a href="{% url 'resend_activation_link_region' region_slug=region.slug user_id=user.pk %}" class="hover:underline text-blue-500 hover:text-blue-600">
				{% trans 'Resend activation link' %}
			</a>
```
I think it would be cool to have the x-icon together with the resend-link...